src/wallet/wallet.cpp

```diff
@@ -1924,33 +1924,25 @@ std::set<uint256> CWalletTx::GetConflicts() const
     return result;
 }
 
+CAmount CWalletTx::GetCachable(CachableAccount& acct, bool credit, const isminefilter& filter, bool recalculate) const
```

Comment:
I know that various people have different feelings about this practice, but invocations would be more readable and less error-prone if `bool credit` were an `enum class` of `CREDIT` or `DEBIT`.

(I feel less strongly about `recalculate` because it's unambiguous which direction is "off" vs "on", and it defaults to "off".)

In fact, better still -- since any given `CachableAccount` is either credit or debit, why not have the account carry the flag, and remove the error-proneness of passing it by hand entirely?

Reply:
Great idea, done.